newIDE/app/src/SceneEditor/index.js

```diff
@@ -1281,6 +1281,33 @@ export default class SceneEditor extends React.Component<Props, State> {
             onClose={() => this.editLayoutVariables(false)}
           />
         )}
+        <InfoBar
+          show={this.state.objectsListAdditionalServiceInfoBar.show}
+          identifier={this.state.objectsListAdditionalServiceInfoBar.identifier}
+          message={this.state.objectsListAdditionalServiceInfoBar.message}
+          touchScreenMessage={
+            this.state.objectsListAdditionalServiceInfoBar.touchScreenMessage
+          }
+        />
+        <InfoBar
+          show={this.state.instancesEditorAdditionalServiceInfoBar.show}
+          identifier={
+            this.state.instancesEditorAdditionalServiceInfoBar.identifier
+          }
+          message={
+            <Trans>
+              {this.state.instancesEditorAdditionalServiceInfoBar.message}
+            </Trans>
+          }
+          touchScreenMessage={
+            <Trans>
+              {
```

Comment:
`<InfoBar>` wasn't really working in either `<ObjectsList>` or `InstancesEditor`, I decided to have them in `SceneEditor` instead, which I guess is consistent with the other `InfoBars` as well, is it okay?

Reply:
I think it's ok, though I recommend in the state to store only one set of details (we only care about showing one of these info bar at a single time anyway), to avoid polluting the state with both the one for the object and the instance.